index.php

```diff
@@ -19,7 +20,31 @@
             <h4 class="font-style-micro font-size-small margin-bottom-small text-align-center"><?php the_time('d F Y'); ?></h4>
             <h3 class="margin-bottom-small text-align-center"><?php the_title(); ?></h3>
 
-            <?php the_excerpt(); ?>
+    <?php
```

Comment:
I still can't find where else is this same markup implemented. I went thru the designs and couldn't find it. The functionality in Submit is the same but I think is different markup.

If you could point me to which templates reuse this I'd gladly make it into a function, otherwise I can't even tell what gets reused.